test/gtest/ucs/test_bitops.cc

```diff
@@ -91,111 +91,51 @@ UCS_TEST_F(test_bitops, ptr_ctz) {
 }
 
 UCS_TEST_F(test_bitops, is_equal) {
-    uint8_t buffer1[20] = {0};
-    uint8_t buffer2[20] = {0};
-
-    ASSERT_TRUE(ucs_bitwise_is_equal(buffer1, buffer2, 0));
-    ASSERT_TRUE(ucs_bitwise_is_equal(buffer1, buffer2, 1));
-    ASSERT_TRUE(ucs_bitwise_is_equal(buffer1, buffer2, 8));
-    ASSERT_TRUE(ucs_bitwise_is_equal(buffer1, buffer2, 64));
-    ASSERT_TRUE(ucs_bitwise_is_equal(buffer1, buffer2, 65));
-    ASSERT_TRUE(ucs_bitwise_is_equal(buffer1, buffer2, 128));
-    ASSERT_TRUE(ucs_bitwise_is_equal(buffer1, buffer2, 130));
-    ASSERT_TRUE(ucs_bitwise_is_equal(buffer1, buffer2, 159));
-    ASSERT_TRUE(ucs_bitwise_is_equal(buffer1, buffer2, 160));
+    uint8_t buffer1[20]         = {0};
+    uint8_t buffer2[20]         = {0};
+    std::vector<int> indices    = {0, 1, 8, 64, 65, 128, 130, 159, 160};
+
+    auto check_bitwise_equality = [](const uint8_t* buffer1,
```

Comment:
can it be just a static method in test_bitops cleass?

Reply:
done